### PR TITLE
zh-translation: /docs/examples/microservices-istio/single/index.md

### PR DESCRIPTION
**Pull request by GuangmingLuo**
Ref: servicemesher/istio-official-translation/issues/1721

### CODE REVIEW

**Comment by GuangmingLuo** [comment]
Depend on this issue: servicemesher/istio-official-translation/issues/1674

**Comment by yuxiaobo96** [comment]
@GuangmingLuo 
测试报错：
1. `/zh/docs/examples/microservices-istio/package-service`下的文档未中文翻译，不存在
2. 链接超时

File: content/zh/docs/examples/microservices-istio/single/index.md [hunk 1]
```diff
@@ -51,8 +49,7 @@ weight: 10
         - 状态码
 
     {{< tip >}}
-    在 Node.js 中, web 服务器的功能嵌入在应用程序的代码中。 一个 Node.js
-    web 应用程序作为一个独立进程运行。
+    在 Node.js 中, web 服务器的功能嵌入在应用程序的代码中。 一个 Node.js web 应用程序作为一个独立进程运行。
```
Comment (yuxiaobo96) [inline content/zh/docs/examples/microservices-istio/single/index.md]:
```suggestion
    在 Node.js 中，web 服务器的功能嵌入在应用程序的代码中。 一个 Node.js web 应用程序作为一个独立进程运行。
```

File: content/zh/docs/examples/microservices-istio/single/index.md [hunk 2]
```diff
@@ -51,8 +49,7 @@ weight: 10
...
     {{< /tip >}}
 
 1. Node.js 应用程序是用 JavaScript 编写的， 这意味着没有显式编译步骤。 相反，它们
```
Comment (yuxiaobo96) [inline content/zh/docs/examples/microservices-istio/single/index.md]:
```suggestion
1. Node.js 应用程序是用 JavaScript 编写的，这意味着没有显式编译步骤。相反，它们
```

**Pull request merged**